src/Components/Buttons/ResetSort.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ResetSortProps } from '../../types/buttons/ResetSortProps';
+import { SortType } from '../../Enums/SortType';
+
+export const ResetSort: React.FC<ResetSortProps> = ({
+  setSortBy,
+  setIsReversed,
+}) => {
+  return (
+    <button
+      type="button"
+      className="button is-danger is-light"
+      onClick={() => {
+        setSortBy(SortType.INITIAL);
+        setIsReversed(false);
+      }}
+    >
+      Reset
+    </button>
+  );
+};
```

Comment:
[UX] - The 'ResetSort' button should visually indicate if it is the active action. Consider adding a style change or an attribute that marks the button as active when the sort type is 'SortType.INITIAL' and the list is not reversed.